x-pack/test/functional/apps/canvas/custom_elements.ts

```diff
@@ -23,10 +22,6 @@ export default function canvasCustomElementTest({
     this.tags('skipFirefox');
 
     before(async () => {
-      // init data
-      await esArchiver.loadIfNeeded('logstash_functional');
-      await esArchiver.load('canvas/default');
```

Comment:
I'd prefer to not move the `await esArchiver.load('canvas/default');` from the before method of each test to the before method of the index.  These tests are apparently all passing now but there is a risk that one test suite impacts another one if we don't reset the .kibana index between test suites (which I assume is the `canvas/default` esArchive).

Reply:
agreed